src/main/include/Constants.h

```diff
@@ -170,9 +170,18 @@ constexpr uint8_t kLedAddress = 23;
 // Motor IDs
 namespace CANSparkMaxConstants {
 constexpr int kIntakeSpinnyBoyID = 20;
+constexpr int kVectorSpinnyBoyID = 21;
```

Comment:
should be kIntakeSpinnyBo**i** and kVectorSpinnyBo**i** ;)

Reply:
100%